Blood_Raven.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name" : "Blood Raven",
+    "lore" : "Blood Raven fought valiantly against Diablo in the catacombs beneath Tristram...She was never quite the same afterwards. It is now obvious she brought an evil influence back with her.",
+
+    "health_points"   : 113,
+    "damage"          : 8,
+    "attack_cooldown" : 1.2,
+    "race" : "undead"
+}
```

Comment:
Ez mit keres itt?! Ez a masik brach feladata.

Reply:
Igen, my bad, másik branchnél is volt ebből a félreértésből probléma, a refactorból lett nyitva, az indokolatlan sok commitot is ez eredményezi. Akkor ha kidobálok minden nem ide való dolgot, rendben lesz?

Reply:
Isten orizz, inkabb akkor a targetet tedd at a refactorra.